server/service/microsoft_mdm.go

```diff
@@ -680,14 +680,37 @@ func NewDMClientProvisioningData() mdm_types.Characteristic {
 			newCharacteristic(syncml.DocProvisioningAppProviderID,
 				[]mdm_types.Param{}, []mdm_types.Characteristic{
 					newCharacteristic("Poll", []mdm_types.Param{
-						newParm("NumberOfFirstRetries", syncml.DmClientCSPNumberOfFirstRetries, syncml.DmClientIntType),
-						newParm("IntervalForFirstSetOfRetries", syncml.DmClientCSPIntervalForFirstSetOfRetries, syncml.DmClientIntType),
-						newParm("NumberOfSecondRetries", syncml.DmClientCSPNumberOfSecondRetries, syncml.DmClientIntType),
-						newParm("IntervalForSecondSetOfRetries", syncml.DmClientCSPIntervalForSecondSetOfRetries, syncml.DmClientIntType),
-						newParm("NumberOfRemainingScheduledRetries", syncml.DmClientCSPNumberOfRemainingScheduledRetries, syncml.DmClientIntType),
-						newParm("IntervalForRemainingScheduledRetries", syncml.DmClientCSPIntervalForRemainingScheduledRetries, syncml.DmClientIntType),
-						newParm("PollOnLogin", syncml.DmClientCSPPollOnLogin, syncml.DmClientBoolType),
-						newParm("AllUsersPollOnFirstLogin", syncml.DmClientCSPPollOnLogin, syncml.DmClientBoolType),
+						// AllUsersPollOnFirstLogin - enabled
+						// https://learn.microsoft.com/en-us/windows/client-management/mdm/dmclient-csp#deviceproviderprovideridpollalluserspollonfirstlogin
+						newParm("AllUsersPollOnFirstLogin", "true", syncml.DmClientBoolType),
```

Comment:
I feel it's clearer to have the actual value here instead of a single-use constant that you have to lookup.